mizar/arktos/arktos_service.py

```diff
@@ -52,17 +55,26 @@ def CreatePod(self, request, context):
         param.body['status']['phase'] = request.phase
         param.body['metadata']['tenant'] = request.tenant
         param.extra = {}
+        ep_name = request.name+"-default-{}-eth0".format(request.tenant)
```

Comment:
"default" and "eth0" may be subjected to be variable?

Reply:
Right. The interface part may be anything. However, I noticed that even in a different tenant I was still getting "default". Let me also fix this.

Reply:
Fixed